release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1288,6 +1290,24 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-release-note-dev-console-bug-fixes_{context}"]
 ==== Developer Console
 
+* Previously, if a `PipelineRun` was using a resolver, rerun of that `PipelineRun` was having issue. With this fix, a user can rerun `PipelineRun` if it is using resolver. (link:https://issues.redhat.com/browse/OCPBUGS-45228[*OCPBUGS-45228*])
+
+* Previously, on the *Edit of Deployment* or *DeploymentConfig* in form view added duplicate `ImagePullSecrets`. With this update, editing the form does not add duplicate entries. (link:https://issues.redhat.com/browse/OCPBUGS-45227[*OCPBUGS-45227*])
+
+* Previously, when you searched on the `OperatorHub` or another catalog, you would experience periods of latency between each key press. With this update, the input on the catalog search bars are debounced. (link:https://issues.redhat.com/browse/OCPBUGS-43799[*OCPBUGS-43799*])
+
+* The *Getting started* section with resources was converted to a collapse and expandable section from {product-title} 4.16, but there was no option to close the *Getting started* section. With this change, user can close the *Getting started* section. (link:https://issues.redhat.com/browse/OCPBUGS-38860[*OCPBUGS-38860*])
```

Comment:
Also, the Getting started resources tile is in the Administrator perspective, but listed here in the Developer Console section. Is that correct?
```suggestion
* Previously, there was no option to close the *Getting started resources* section. With this change, user can close the *Getting started resources* section. (link:https://issues.redhat.com/browse/OCPBUGS-38860[*OCPBUGS-38860*])
```